README.md

```diff
@@ -38,6 +38,24 @@ Add new dependencies in the usual way:
 npm install <new-package>
 ```
 
+### Database Setup
+
+#### Database Server Setup
+
+A SQL Server database server has been set up to run when the appeals service api is running.
+
+##### Azure Data Studio
+
+[Azure Data Studio](https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio) can be used as a database client, to create and monitor the database contents.
```

Comment:
Looks good. There's also a vscode extension that is handy: https://marketplace.visualstudio.com/items?itemName=ms-mssql.mssql